### PR TITLE
Dutch specs localized and filtered

### PR DESCRIPTION
**Pull request by LionbridgeCS**
Review of pull request  #1594

### CODE REVIEW

**Comment by tellarin** [comment]
@LionbridgeCS Please don't close a PR with ongoing reviews to create a new one, unless that's discussed on the original PR for clarity. Otherwise it gets much harder to track if review comments were addressed.
If a commit is mistakenly pushed to a PR, you can update the PR reverting it.

File: Specs/DateTime/Dutch/HolidayParser.json [hunk 1]
```diff
@@ -303,5 +303,345 @@
...
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "dotnet",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "Pasen",
+        "Type": "date",
+        "Value": {
+          "Timex": "",
```
Comment (tellarin) [inline Specs/DateTime/Dutch/HolidayParser.json]: @iMicknl had asked a question about this, right?
@LionbridgeCS, could you please reply here?

Comment (LionbridgeCS) [reply Specs/DateTime/Dutch/HolidayParser.json]: This one was localized from the English counter part which does not have a Timex for Easter.

**Comment by LionbridgeCS** [comment]
@tellarin Thank you for the directive, noted.

File: Specs/DateTime/Dutch/HolidayParser.json [hunk 2]
```diff
@@ -303,5 +303,345 @@
...
+          },
+          "PastResolution": {
+            "date": "2018-05-01"
+          }
+        },
+        "Start": 29,
+        "Length": 7
+      },
+      {
+        "Text": "Int'l Workers Day",
```
Comment (iMicknl) [inline Specs/DateTime/Dutch/HolidayParser.json]: This string doesn't exist in `Input` text, so this resolution should be removed.

File: Specs/DateTime/Dutch/HolidayParser.json (hunk 1 above)
Comment (iMicknl) [reply Specs/DateTime/Dutch/HolidayParser.json]: Thanks, had a look at English and Easter is indeed a date that can't have a Timex.

**Pull request merged**